components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -1563,6 +1563,19 @@ impl Animatable for InnerMatrix2D {
     }
 }
 
+impl ComputeSquaredDistance for InnerMatrix2D {
+    #[inline]
+    fn compute_squared_distance(&self, other: &Self) -> Result<SquaredDistance, ()> {
+        // If possible, we should convert the InnerMatrix2D into types with physical meaning.
+        // Therefore, we compute the squared distance from each matrix item, and this makes the
+        // result different from that in Gecko if we have skew/shear factor in the ComputedMatrix.
```

Comment:
Shear is used in Gecko. I think I should remove this word to avoid confusing people.